geospaas_processing/copiers.py

```diff
@@ -31,8 +31,13 @@ def write_flag_file(type_in_flag_file, source_path, dataset, destination_filenam
         """
         with open(destination_filename + ".flag", "w") as flag_file:
             flag_file.write(f"type: {type_in_flag_file}" + os.linesep)
+            flag_file.write(f"entry_id: {dataset.entry_id}" + os.linesep)
+            flag_file.write(f"entry_title: {dataset.entry_title}" + os.linesep)
+            flag_file.write(f"source: {dataset.source}" + os.linesep)
+            flag_file.write(f"data_center: {dataset.data_center}" + os.linesep)
             for urlname in dataset.dataseturi_set.exclude(service=LOCAL_FILE_SERVICE):
-                flag_file.write(f"url: {urlname.uri}" + os.linesep)
+                flag_file.write(f"- url: {urlname.uri}" + os.linesep)
+            flag_file.write(f"summary: {dataset.summary}" + os.linesep)
```

Comment:
Maybe build the string and then write it in one go, rather than calling `write()` multiple times.

Reply:
Done in 4b67c5b